docs/src/1_Mechanics/01_PlayerGuide_Full.md

```diff
@@ -291,6 +290,22 @@ They draw a TC of 3 of Diamonds, permitting a card between A and 6. Jonas draws
 Spades with a +1 Modifier to Detection and reports a 7, still outside the DR. Xena would 
 successfully convince Jonas of something in this scenario. 
 
+#### Saves
+
+Saves occur when a character is attempting to resist or shrug off a negative effect.
+Certain Powers, effects, or encounters may force a character to make a Save. When a
+character makes a Save, they draw a card against their own TC and use their Attribute
+Modifier for the relevant domain. A character cannot choose to use a different
+Attribute.
+
+A Save will have a DR determined by the initiator's Skill Modifier used to initiate the
+Save. The DR for a Save is 3 minus half of the initiator's relevant modifier, to a
+minimum of 0. 
+
+Saves may have different effects depending on a success or failure. This can sometimes
+mean a successful Save still results in taking damage or a negative effect. Reference a
+specific Power description for details on the effects.
```

Comment:
```suggestion
When a character attempts to resist or shrug off a negative effect, they make a Save.
This could be caused by a Power, Status Condition, or a environmental effect. When a
character makes a Save, they draw a card against their own TC and use their relevant Attribute
Modifier, with a DR set by the initiating effect. If the effect is a Power, the DR is
3 minus half of the initiator's relevant Skill modifier. 
Saves may still have consequences even on a success, as specified by the initiating effect.
```